web-registry/src/components/templates/ProjectDetails/ProjectDetails.tsx

```diff
@@ -225,6 +225,8 @@ function ProjectDetails(): JSX.Element {
             ? () => setIsBuyFlowStarted(true)
             : () => setDisplayErrorBanner(true)
         }
+        onChainProjectId={onChainProjectId as string}
+        projectName={metadata?.['schema:name']}
```

Comment:
this how we pass in the values to the sticky nav buy button action for the project page.